internal/pkg/commands/cmd_runner_unix.go

```diff
@@ -0,0 +1,96 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type UnixCMDRunner struct {
+	RunWithSudo bool
+}
+
+func NewCMDRunner(options CMDRunnerOptions) CommandRunner {
+	return &UnixCMDRunner{
+		RunWithSudo: options.RunAsAdmin,
+	}
+}
+
+func (cr *UnixCMDRunner) BuildDockerComposeUpCMD(options DockerComposeUpOptions) Command {
+	servs := strings.Join(options.Services, " ")
+	command := fmt.Sprintf("docker-compose -f %s up -d %s", options.Path, servs)
+	return Command{Cmd: command}
+}
+
+func (cr *UnixCMDRunner) BuildDockerPSCMD(options DockerPSOptions) Command {
+	command := "docker ps"
+	if options.All {
+		command += " -a"
+	} else {
+		log.Info(`Command "docker ps" built without the "--all" flag.`)
+	}
+	return Command{Cmd: command}
+}
+
+func (cr *UnixCMDRunner) BuildDockerComposePSCMD(options DockerComposePsOptions) Command {
+	servs := ""
+	if options.Services {
+		log.Info(`Command "docker-compose ps" built with "--service" flag.`)
```

Comment:
Same Debug topic. The thing is that the full command is displayed outside with a log.Info, so I see little point in being redundant in these cases with an Info log too. Is good to log this, but what do you think about changing this to Debug?